lib/Reader/readerir.cpp

```diff
@@ -4094,7 +4094,21 @@ IRNode *GenIR::loadField(CORINFO_RESOLVED_TOKEN *ResolvedToken, IRNode *Obj,
   // call the helper routine; we can't just load the address
   // and do a load-indirect off it.
   if (FieldInfo.fieldAccessor == CORINFO_FIELD_INSTANCE_HELPER) {
-    throw NotYetImplementedException("LoadField via helper");
+    handleMemberAccess(FieldInfo.accessAllowed, FieldInfo.accessCalloutHelper);
+    Value *Result;
```

Comment:
You can move this down and type it as IRNode * so that you don't need a cast later.